src/apps/Altinn.AccessManagement/src/Altinn.AccessMgmt.Core/Utils/Mappers/DtoMapperConnectionQuery.cs

```diff
@@ -8,10 +8,10 @@ public partial class DtoMapper : IDtoMapper
 {
     public static List<ConnectionDto> ConvertToOthers(IEnumerable<ConnectionQueryExtendedRecord> connections)
     {
-        var result = connections.GroupBy(res => res.ToId).Select(c =>
+        var result = connections.Where(t => t.Reason == ConnectionReason.Assignment || t.Reason == ConnectionReason.Delegation || t.Reason == ConnectionReason.RoleMap).GroupBy(res => res.ToId).Select(c =>
```

Comment:
Dette er da grunnen til at jeg mener vi bør undersøke å implementere direksjonelle spørringer.

For nå henter vel spørringen masse data som må forkastes i etterkant, noe som er en stor risiko for at en utvikler kan glemme/ikke vite om. Som gjør at feil tilgangsdata slipper ut i løsningen.